sdk_container/src/third_party/coreos-overlay/app-emulation/amazon-ssm-agent/amazon-ssm-agent-3.2.985.0-r4.ebuild

```diff
@@ -3,7 +3,6 @@
 EAPI=7
 
 COREOS_GO_PACKAGE="${GITHUB_URI}"
-COREOS_GO_VERSION="go1.19"
```

Comment:
If we start building amazon-ssm-agent with Go 1.21, we should probably either bump the version of amazon-ssm-agent to a fairly recent one coupled with Go 1.21, or at least run CI for AWS without updating amazon-ssm-agent.

Actually we should run a full CI for all cloud providers, not only qemu, because bumping Go version is one of the most invasive changes in Flatcar.